tasks/npm.yml

```diff
@@ -33,13 +34,12 @@
           - "{{ tmp_dir }}:/status"
         state: started
         working_dir: "{{ active_deploy_dir }}"
-        command: /bin/ash -c "{{django_stack_npm_install_cmd}}; touch /status/npm.done"
+        command: /bin/ash -c "{{ django_stack_npm_install_cmd }}"
         user: "{{ 'node' if (lookup('pipe','id -u') == '1000') else omit }}"
+        # Block until command is finished; report failure on error
+        detach: no
         recreate: "yes"
-
-    - name: Wait for node logic to finish up
-      wait_for:
-        path: "{{ tmp_dir }}/npm.done"
+        keep_volumes: no
```

Comment:
what about destroying `node_modules` folder everytime for `ci`? Or having a flag to do that ? That would really ensure that everything is getting rebuilt fresh each time in a local environment